django/utils/formats.py

```diff
@@ -172,18 +172,19 @@ def localize(value, use_l10n=None):
     If use_l10n is provided and is not None, that will force the value to
     be localized (or not), overriding the value of settings.USE_L10N.
     """
+    if isinstance(value, six.string_types):
+        return value
     if isinstance(value, bool):
         return mark_safe(six.text_type(value))
-    elif isinstance(value, (decimal.Decimal, float) + six.integer_types):
```

Comment:
I'm not aware on any format rules/best practices around using if/elif in cases like this. It's my personal preference to only use elif when it is actually necessary for the logic to work correctly. (Which is not the case in this instance). I'll revert to using elif if that's more in line with the rest of Django.